src/status_im/communities/core.cljs

```diff
@@ -780,3 +780,24 @@
                                 :community-id community-id
                                 :public-key   public-key
                                 :role-id      role-id})]})
+
+(rf/defn mute-chat-failed
+  {:events [:community/mute-community-failed]}
+  [{:keys [db]} community-id muted? error]
+  (log/error "mute community failed" community-id error)
+  {:db (assoc-in db [:communities community-id :muted] (not muted?))})
+
+(rf/defn mute-community-successfully
+  {:events [:community/mute-community-successful]}
+  [_ community-id]
+  (log/debug "muted community successfully" community-id))
+
+(rf/defn set-community-muted
+  {:events [:community/set-muted]}
+  [{:keys [db]} community-id muted?]
+  {:db            (assoc-in db [:communities community-id :muted] muted?)
+   :json-rpc/call [{:method     "wakuext_setCommunityMuted"
+                    :params     [community-id muted?]
+                    :on-error   #(rf/dispatch [:community/mute-community-failed community-id
+                                               muted? %])
+                    :on-success #(rf/dispatch [:community/mute-community-successful community-id])}]})
```

Comment:
@jo-mut, quick one. Often the codebase tries to use the following convention:

- The success event is named as the keyword in `{:events ...}` with the suffix `success`
- The error event is named as the keyword in `{:events ...}` with the suffix `error`

So in this PR you could go for this suggestion:

```clojure
(rf/defn set-community-muted
  {:events [:community/mute]}
  [{:keys [db]} community-id muted?]
  {:db            (assoc-in db [:communities community-id :muted] muted?)
   :json-rpc/call [{:method     "wakuext_setCommunityMuted"
                    :params     [community-id muted?]
                    :on-error   #(rf/dispatch [:community/mute-error community-id
                                               muted? %])
                    :on-success #(rf/dispatch [:community/mute-success community-id])}]})
```